src/frontend/Sider/Journalføring/Felles/utils.ts

```diff
@@ -104,7 +104,19 @@ export const alleBehandlingerErFerdigstiltEllerSattPåVent = (behandlinger: Beha
                 behandling.type === BehandlingType.REVURDERING)
     );
 
-export const utledBehandlingstype = (tidligereBehandlinger: Behandling[]): BehandlingType => {
+export const utledBehandlingstype = (
+    tidligereBehandlinger: Behandling[],
+    journalføringsårsak: Journalføringsårsak
+): BehandlingType => {
+    const journalføringsårsakErEnKlagesak = (journalføringsårsak: Journalføringsårsak): boolean => {
+        return (
+            journalføringsårsak === Journalføringsårsak.KLAGE ||
+            journalføringsårsak === Journalføringsårsak.KLAGE_TILBAKEKREVING
+        );
+    };
+    if (journalføringsårsakErEnKlagesak(journalføringsårsak)) {
+        return BehandlingType.KLAGE;
+    }
```

Comment:
Tenker en funksjon er litt ovekill her.
```suggestion
if (
        [Journalføringsårsak.KLAGE, Journalføringsårsak.KLAGE_TILBAKEKREVING].includes(
            journalføringsårsak
        )
    ) {
        return BehandlingType.KLAGE;
    }}
```